extensions/adobe/experience/customerJourneyManagement/message-export-schema.json

```diff
@@ -0,0 +1,188 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/experience/customerJourneyManagement/message-export",
+  "title": "Adobe CJM ExperienceEvent - Message Export",
+  "description": "Schema for storing the rendered message content for outbound channels in Adobe Journey Optimizer",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/experienceevent"],
+  "allOf": [
+    {
+      "$ref": "https://ns.adobe.com/xdm/data/record"
+    },
+    {
+      "properties": {
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageExport/identityMap": {
+          "title": "Identity Map",
+          "description": "Map containing identifiers for the profile across different identity namespaces",
+          "$ref": "https://ns.adobe.com/xdm/context/identitymap"
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageExport/messageExecution": {
+          "type": "object",
+          "title": "Adobe CJM ExperienceEvent - Message Execution Details",
+          "description": "Message Execution details for the Adobe CJM ExperienceEvent.",
+          "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/messageexecution"
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageExport/messageProfile": {
+          "type": "object",
+          "title": "Adobe CJM ExperienceEvent - Message Profile Details",
+          "description": "Message Profile details for the Adobe CJM ExperienceEvent.",
+          "$ref": "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile"
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageExport/message": {
+          "type": "object",
+          "title": "Message Details",
+          "description": "Details of the rendered message",
+          "properties": {
+            "messageDetails": {
+              "type": "object",
+              "title": "Message Details",
+              "description": "Contains the specific content details for the message, varying by channel (e.g., email or SMS).",
+              "oneOf": [
```

Comment:
I am not sure if oneOf will work. What are you expecting the validator to validate?